Sources/UIApplication+handleSDLEvents.swift

```diff
@@ -14,6 +14,14 @@ extension UIApplication {
         var e = SDL_Event()
 
         while SDL_PollEvent(&e) == 1 {
+
+            #if os(Android)
+            if let uiEvent = UIEvent.from(e) {
+                sendEvent(uiEvent)
+                break
+            }
+            #endif
```

Comment:
its not clear why we send send events for android only

Reply:
I'll write a comment, thanks